.github/workflows/go-check-config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "gogenerate": true
```

Comment:
This means that we'll generate the mocks on CI as well, and fail the build when there's a mismatch. So there's no way we can forget updating the mock when updating deps.

Reply:
nice!